spark/src/test/scala/org/apache/comet/parquet/ParquetReadFromS3Suite.scala

```diff
@@ -76,4 +91,32 @@ class ParquetReadFromS3Suite extends CometS3TestBase with AdaptiveSparkPlanHelpe
     assertCometScan(df)
     assert(df.first().getLong(0) == 499500)
   }
+
+  test("write and read encrypted parquet from S3") {
```

Comment:
I don't know if this test exercises the changes to normalizing the path or not, but the test looks good

Reply:
It did fail on main, FWIW.